.github/workflows/rector_ci.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
 
             ## First run Rector - here can't be --dry-run !!! it would stop the job with it and not commit anything in the future
             -   run: bin/rector rectify ${{ matrix.directories }} --config rector-ci.php --ansi --no-progress-bar
-            -   run: vendor/bin/ecs check --match-git-diff --fix --ansi
+           # -   run: vendor/bin/ecs check --match-git-diff --fix --ansi
```

Comment:
It seems the addition `@var` on `[ci-review] Rector Rectify` related with ecs command, trying disable it.